eng/pipelines/runtime.yml

```diff
@@ -283,6 +281,33 @@ jobs:
           eq(dependencies.checkout.outputs['SetPathVars_installer.containsChange'], true),
           eq(variables['isFullMatrix'], true))
 
+#
+# Build the whole product using Mono and run tests
+#
+- template: /eng/pipelines/common/platform-matrix.yml
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+    buildConfig: Release
+    runtimeFlavor: mono
+    platforms:
+    - iOS_x64
```

Comment:
I guess we should leave iOS where it was before merging for this PR. Maybe we should remove the iOS specific changes here.